webpack_loader/loader.py

```diff
@@ -76,8 +76,12 @@ def get_bundle(self, bundle_name):
                 )
 
         if assets.get('status') == 'done':
-            chunks = assets['chunks'][bundle_name]
-            return self.filter_chunks(chunks)
+            chunks = assets['chunks']
+            try:
+                bundle_chunks = chunks[bundle_name]
+            except KeyError:
+                raise WebpackError('Cannot resolve bundle {}.'.format(bundle_name))
```

Comment:
`WebpackError` is used to represent errors that were raised by webpack itself. This should use a new exception and how about,

```python
chunks = assets['chunks'].get(bundle_name, None)
if chunks is None:
     raise WebpackBundleLookupError("...")
```